examples/flask/airbrake_middleware.py

```diff
@@ -0,0 +1,43 @@
+import sys
+import datetime
+from flask import g, request
+from argparse import ArgumentParser
+
+sys.path.append('../../pybrake')
+import routes
+from utils import logger
+
+parser = ArgumentParser()
+parser.add_argument("-project_id", dest="project_id", help="airbrake project ID")
+parser.add_argument("-project_key", dest="project_key", help="airbrake project key")
+parser.add_argument("-host", dest="host", help="airbrake host")
+
+args = parser.parse_args()
+
+notifier = routes.AirbrakeNotifier(args.project_id, args.project_key, args.host)
```

Comment:
this should use notifier from https://github.com/airbrake/pybrake/blob/master/pybrake/notifier.py#L39